types/lighthouse-stack-packs/index.d.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+declare module 'lighthouse-stack-packs/packs/amp' {
```

Comment:
I didn't want to add these into the `lighthouse-stack-packs` as it adds extra complexity for contributors.

I do think we might want to create a script that generates these definitions. Or at least a script that checks if this file contains all pack definitions.

Reply:
We don't need a script for this. It's super easy to do manually and we don't update this often.

I'm not sure I agree with keeping this in `lighthouse` to avoid complexity. Most contributions won't be _adding_ frameworks. And if one does, then it's rather straightforward. There are other more complex things that must be done to add a framework, such as changes to `js-library-detector`.